src/js/components/ListItem/ListItem.js

```diff
@@ -5,42 +5,41 @@ import style from './ListItem.less';
 import Button from '../Button';
 import styles from './styles';
 
-const ListItem = (props) => {
-  const { classes } = props;
-  return (
-    <Paper className={classes.root}>
-      <div className={style.first}>
-        <h5 className={style.listTitle}>
-          {props.title}
-        </h5>
-        <div className={style.group}>
-          <span className={style.timing}>
-            {props.timing}
-          </span>
-          <span className={style.date}>
-            {props.date}
-          </span>
-          <Button
-            color="primary"
-            variant="outlined"
-          >
-            Edit
-          </Button>
-        </div>
-      </div>
-      <div className={style.second}>
-        <span>
-          {props.text}
+const ListItem = ({
+  title, timing, date, text, classes
+}) => (
+  <Paper className={classes.root}>
+    <div className={style.first}>
```

Comment:
duplications again and you import styles but get properties from style? is it work correctly?

Reply:
here style and styles are different)
style is imported from ListItem.js, and styles from styles.js (this is mui styles)

Reply:
fixed)